### DISCUSSION TITLE
How to overwrite `strong` / `b` font weight?

### DISCUSSION THREAD

**Discussion by MickL** [tailwindlabs/tailwindcss#13513] [Help]
How do I change the default font-weight of `strong` and `b` ? Tailwind sets it to `bolder` and all I can do is add a second style in my personal `tailwind.css`.

Sorry if it is obvious but I couldnt find this in the docs or by google.

**Comment by wongjn** [comment]
You could add a rule in your input CSS like:

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
  strong, b {
    font-weight: <your desired value>;
  }
}
```

Or have a plugin add the rule:

```js
module.exports = {
  // …
  plugins: [
    require('tailwindcss/plugin')(({ addBase }) => {
      addBase({
        'strong, b': {
          fontWeight: 'your desired value',
        },
      });
    }),
  ],
};
```

**Reply by MickL** [reply]
This is what I did but it adds another css rule instead of changing the original one.

<img width="329" alt="Bildschirmfoto 2024-04-12 um 13 50 25" src="https://github.com/tailwindlabs/tailwindcss/assets/5103210/cc2aee5c-70f0-4785-a511-fe3ed345d5a3">

**Reply by wongjn** [reply]
There is no API surface within Tailwind to change the existing rule directly. You could:
- [Disable preflight](https://tailwindcss.com/docs/preflight#disabling-preflight) and run your own version with modified CSS rules.
- Have a PostCSS plugin that runs after Tailwind that modifies the CSS rule, if you are running Tailwind via PostCSS.

**Reply by MickL** [reply]
Maybe this could be considered as a feature request? :)

**Reply by MickL** [reply]
Any update on this with Tailwind 4 @wongjn ? I see main.css sets:

```css
b, strong {
    font-weight: bolder;
}
```

Any way to change this without overwriting it?

<img width="413" alt="Bildschirmfoto 2025-01-28 um 15 16 03" src="https://github.com/user-attachments/assets/7a28cf24-ca36-4e4a-93b0-369da63635e9" />

I created a feature request: #15970
